packages/cosmic-swingset/Makefile

```diff
@@ -173,7 +173,7 @@ compile-gyp:
 install: go.sum
 	go install -v $(MOD_READONLY) $(BUILD_FLAGS) ./cmd/ag-cosmos-helper
 	bindir="$${GOBIN-$${GOPATH-$$HOME/go}/bin}"; \
-	  ln -sf "$$PWD/lib/ag-chain-cosmos" "$$PWD/bin/ag-nchainz" "$$bindir/"
+	  ln -sf "$$PWD/bin/ag-chain-cosmos" "$$PWD/bin/ag-nchainz" "$$bindir/"
```

Comment:
I saw two other places that reference `.../lib/ag-chain-cosmos`: the cosmic-swingset README (`ln` instructions), and `scripts/chain-entry.sh`. Do those need to be updated too?

Reply:
Good catch!  I updated even more references now.